Specifications/Services/ReverseCall/for_PingedConnection/when_pings_are_requested_every_12_seconds/and_next_message_is_recieved_after_36_seconds.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Dolittle.Runtime.Services.ReverseCalls.for_PingedConnection.given;
+using Machine.Specifications;
+
+namespace Dolittle.Runtime.Services.ReverseCalls.for_PingedConnection.when_pings_are_requested_every_12_seconds
+{
+    public class and_next_message_is_recieved_after_36_seconds : given.all_dependencies
```

Comment:
```suggestion
    public class and_next_message_is_received_after_36_seconds : given.all_dependencies
```

Typo, filename should also be fixed